setup.py

```diff
@@ -49,4 +49,5 @@
     py_modules=['arff'],
     package_data={'':['README.rst', 'CHANGES.rst', 'LICENSE']},
     test_suite='tests',
+    install_requires = ['python-dateutil'],
```

Comment:
Unfortunately, this is a major issue. Currently, the package contains no dependencies and advocates:

    Read and write ARFF files using python built-in structures, such dictionaries and lists

Therefore I am very hesitant to add in a new dependency and it would be great if you can find a python built-in replacement for this package. From my understanding a first step would be to only parse a datestring such as `yyyy-MM-dd'T'HH:mm:ss` according to the [WEKA docs](http://weka.wikispaces.com/ARFF%20%28stable%20version%29).

Reply:
How about providing a date_parser parameter to load functions that would handle the case of no format string. You could support 'dateutil' or callable. Default could be str?